scripts/setup-adapters.sh

```diff
@@ -33,32 +33,21 @@ function install_aws_deps {
   cmake_install -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DBUILD_SHARED_LIBS:BOOL=OFF -DMINIMIZE_SIZE:BOOL=ON -DENABLE_TESTING:BOOL=OFF -DBUILD_ONLY:STRING="s3;identity-management"
   # Dependencies for S3 testing
   # We need this specific version of Minio for testing.
-  if [[ "$OSTYPE" == linux-gnu* ]]; then
-    wget https://dl.min.io/server/minio/release/linux-amd64/archive/minio-20220526054841.0.0.x86_64.rpm
-    if yum list installed minio >/dev/null 2>&1;then
-      if  prompt "Do you want to replace the installed Minio with version minio-20220526054841.0.0.x86_64.rpm needed for S3 tests?"; then
-        rpm -i --replacepkgs minio-20220526054841.0.0.x86_64.rpm
-      else
-        echo "Minio version minio-20220526054841.0.0.x86_64.rpm needed for S3 tests is not installed."
-      fi
-    else
-      rpm -i minio-20220526054841.0.0.x86_64.rpm
-    fi
-    rm minio-20220526054841.0.0.x86_64.rpm
+  local MINIO_ARCH=$MACHINE
```

Comment:
nit: Maybe at some point in future we should move these dependencies for adapters inside respective setup-mac/ubuntu/ etc scripts so that this one script doesnt have to have this kind of logic (and also for making things consistent).

Reply:
Make sense. Filed issue https://github.com/facebookincubator/velox/issues/10413